docs(grid): Revamp Grid Loading animation article (#2486)
telerik/blazor-docs: components/grid/loading-animation.md

# Loading Animation

> The Grid can show a loading animation to indicate data operations that take more than 600ms to complete. The indicator appears as a loading sign over the Blazor Data Grid. The loading animation improves user experience with a visual hint that the requested action is still executing. The feature can prevent repetitive user actions.

Reviewer: ```suggestion
You can show a loading animation for Grid data operations that take more than 600ms to complete. This improves the user experience with a visual hint that the requested action is still running and might help prevent repetitive user actions.

The animation appears as a loading indicator over the Blazor Data Grid.
```

## Basics

> The Grid `EnableLoaderContainer` parameter determines if the component will show a built-in LoaderContainer for long operations. The loading animation is enabled by detault. The data operations that trigger the loading animation include:

Reviewer: ```suggestion
The Grid `EnableLoaderContainer` parameter determines if the component will show a built-in LoaderContainer for long-running operations. The loading animation is enabled by default. The data operations that trigger the loading animation include:
```

* [Paging]({%slug components/grid/features/paging%})
* [Filtering]({%slug components/grid/filtering%})
* [Sorting]({%slug components/grid/features/sorting%})
* [Grouping]({%slug components/grid/features/grouping%}) 
* [Expanding groups with load-on-demand]({%slug grid-group-lod%})
* [Creating, deleting or editing records]({%slug components/grid/editing/overview%})

## Show LoaderContainer on Initial Load

> The Grid will not display a loading animation during its initial rendering and data load. The component cannot know when or even if data will be provided to it, especially when using the Grid `Data` parameter. Initial automatic loading sign can either show indefinitely, or it could prevent the user from altering any saved Grid state (such as changing filters). If you want to display a loading animation on initial load, you can use a [LoaderContainer component]({%slug loadercontainer-overview%}). See the example below or the [Grid Loading Animation Live Demo](https://demos.telerik.com/blazor-ui/grid/loading-animation).

Reviewer: ```suggestion
The Grid does not display a loading animation during its initial rendering and data load. The component cannot know when or even if data will be provided to it, especially when using the Grid `Data` parameter. An initial automatic loading sign can either show indefinitely, or it could prevent the user from altering any saved Grid state (such as changing filters).

If you want to display a loading animation on initial load, you can use a [LoaderContainer component]({%slug loadercontainer-overview%}). See the example below or the [Grid Loading Animation Live Demo](https://demos.telerik.com/blazor-ui/grid/loading-animation).
```
